packages/scandipwa/src/component/Breadcrumbs/Breadcrumbs.component.js

```diff
@@ -65,6 +65,7 @@ export class Breadcrumbs extends PureComponent {
             || location.pathname === appendWithStoreCode('/')
             || location.pathname.match(appendWithStoreCode('/account'))
             || location.pathname === '/'
+            || (breadcrumbs.length === 1 && breadcrumbs[0].name === 'Homepage')
```

Comment:
What if string will be translated in breadcrumbs and it will not be 'Homepage' ?  Basically right now it is hardcode, try to not depend on checking strings that are shown on frontend to customer, they are most lucky translated.